src/styles/sidebar/components/NotebookResultCount.scss

```diff
@@ -8,7 +8,10 @@
   line-height: 1;
 
   & h2 {
-    font-weight: bold;
+    em {
```

Comment:
Any reason not to just use a `<b>` or `<strong>` tag here?

Reply:
Nooooo, darn it. What was I thinking?